README.md

```diff
@@ -1 +1,45 @@
-# Leaderboard
\ No newline at end of file
+# Project Name
+
+Leaderboard
+
+# PR
+
+1-Add items to the Leaderboard
+2-send Add items to an API
+![screenshot](./Capture.PNG)
+
+## Built With
+
+- Html
+- css
+- JavaScript
+
+## live demo :
+-not yet
+
+## Getting Started
+
+-open the index.html file in chrome and see the Leaderboard
+
+### Setup
+
+-click the code button in the repo and then download the files as a zip file
+
+#### To get a local copy up and running follow these simple example steps.
+
+- Click the green `Code` button on the repo and copy link
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/Leaderboard`
+- On the terminal, navigate into the directory like so: `cd Leaderboard`
+
+### Prerequisites
+
+Chrome or any explorer
+
+## Authors
+
+👤 **Meqdam Al-qudah**
+
+- [GitHub](https://github.com/MeqdamAlqudah)
+- [Twitter](https://twitter.com/MeqdamQudah)
+- [LinkedIn](www.linkedin.com/in/meqdam-al-qudah-7514a21b5)
```

Comment:
- [ ] Please consider making use of the Microverse [README](https://github.com/microverseinc/readme-template) template and add the section I captured in the image below. This is to ensure we all have a descriptive README and which follows Microverse standards.

<img width="448" alt="chrome_8Qk9tbvkfm" src="https://user-images.githubusercontent.com/81468675/146457097-8d1501b4-a6b0-4165-a18b-ea36e9a7f356.png">